wpilibc/src/main/native/include/frc/geometry/Rotation2d.h

```diff
@@ -0,0 +1,132 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+namespace frc {
+
+constexpr double kPi = 3.14159265358979323846;
```

Comment:
Should this be in the namespace? Seems slightly odd to have it in the frc namespace. Should probably be private static in the class.

Reply:
Fixed in 52f3a09